recipes/lerc/all/test_package/conanfile.py

```diff
@@ -22,5 +24,9 @@ def build(self):
 
     def test(self):
         if can_run(self):
-            bin_path = os.path.join(self.cpp.build.bindirs[0], "test_package")
+            bin_path = os.path.join(self.cpp.build.bindir, "test_package_cxx")
             self.run(bin_path, env="conanrun")
+            if not is_msvc(self):
```

Comment:
It requires better investigation, otherwise will be broken for users too. Also, it's true the test package for C could be much simpler than the actual one.